app/api/auth/thirdweb/gateCondition.ts

```
@@ -1,14 +1,22 @@
 import { base } from "thirdweb/chains";
-import { client } from "../../../lib/sdk/thirdweb/client";
-import { balanceOf as balanceOfERC721 } from "thirdweb/extensions/erc721";
+import { createThirdwebClient } from "thirdweb";
+import dotenv from 'dotenv';
+
+// Load environment variables from .env file
+dotenv.config();
+
+// Initialize the client using createThirdwebClient
+const client = createThirdwebClient({
+  clientId: process.env.THIRDWEB_ADMIN_PRIVATE_KEY!,  // Use non-null assertion or proper env validation
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)